install_config/http_proxies.adoc

```diff
@@ -279,6 +279,18 @@ MAVEN_ARGS_APPEND=" -s path/to/file"
 ====
 ////
 
+[[using-maven-behind-a-proxy]]
+== Using Maven Behind a Proxy
+
+Using Maven with proxies requires using the `HTTP_PROXY_NONPROXYHOSTS` variable.
+
+See the
+link:https://access.redhat.com/documentation/en-us/red_hat_jboss_middleware_for_openshift/3/html-single/red_hat_jboss_enterprise_application_platform_for_openshift/#configuring_eap_for_openshift[Red
+Hat JBoss Enterprise Application Platform for OpenShift documentation] for
+information for how to configure your {product-title} environment for Red Hat
+JBoss Enterprise Application Platform, including the steps for setting up Maven
+behind a proxy.
+
```

Comment:
Does the user need to do all the steps in the linked section, or just the note that talks about proxies?